README.adoc

```diff
@@ -216,9 +216,18 @@ acceptance tests. They can be run via:
 $ bundle exec rake features
 ---------------------------
 
+You can run a single feature specifying the path to the feature file
```

Comment:
@Preeticp @bexelbie Could you go through the documentation changes here?

I have gone through the code related changes and its LGTM.